cms/djangoapps/contentstore/tests/test_import.py

```diff
@@ -219,26 +219,6 @@ def test_rewrite_reference_list(self):
             conditional_module.show_tag_list
         )
 
-    def test_rewrite_reference(self):
```

Comment:
Similarly to `test_advanced_components_munge_tabs`, should `open_ended` be _replaced_ instead of _removed_?

Do we care that we no longer test this condition?